db/migrate/20200203200855_add_ancestry_to_miq_ae_namespace.rb

```diff
@@ -0,0 +1,31 @@
+require 'ancestry'
+
+class AddAncestryToMiqAeNamespace < ActiveRecord::Migration[5.0]
```

Comment:
If this is for master, we should be generating the migration using rails 5.1.  I think the 5.0 here would still work but going forward, we'll be doing 5.1, hopefully 5.2 soon.